client/lib/posts/utils.js

```diff
@@ -6,8 +6,8 @@
 
 import url from 'url';
 import i18n from 'i18n-calypso';
-import moment from 'moment-timezone';
 import { includes } from 'lodash';
+const moment = i18n.moment;
```

Comment:
Why not `import { moment } from 'i18n-calypso'`? It's used this way at a lot of places in Calypso, so it should work.

Reply:
ah cool! I just didn't know that!